OrbitGl/ThreadTrack.cpp

```diff
@@ -119,11 +119,19 @@ Color ThreadTrack::GetTimerColor(const TimerInfo& timer_info, bool is_selected)
 
   uint64_t address = timer_info.function_address();
   const FunctionInfo* function_info = GOrbitApp->GetCaptureData().GetSelectedFunction(address);
-  CHECK(function_info);
```

Comment:
Same here, is there a way to keep the `CHECK`?

Reply:
Done.

Reply:
yep, done.